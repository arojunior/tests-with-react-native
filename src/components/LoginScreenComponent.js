import React from 'react';
import {
  Container,
  Header,
  Title,
  Content,
  Footer,
  FooterTab,
  Button,
  Left,
  Right,
  Body,
  Icon,
  Text
} from 'native-base';
import LoginFormContainer from '../containers/LoginFormContainer';

const LoginScreenComponent = () => (
  <Container>
    <Header>
      <Left>
        <Button transparent>
          <Icon name="menu" />
        </Button>
      </Left>
      <Body>
        <Title>Header</Title>
      </Body>
      <Right />
    </Header>
    <Content>
      <LoginFormContainer />
    </Content>
    <Footer>
      <FooterTab>
        <Button full>
          <Text>Tests with react-native</Text>
        </Button>
      </FooterTab>
    </Footer>
  </Container>
);

export default LoginScreenComponent;
