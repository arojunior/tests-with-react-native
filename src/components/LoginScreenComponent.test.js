import LoginScreenComponent from './LoginScreenComponent';

describe(`LoginScreenComponent`, () => {
  test(`Snapshot`, () => {
    expect(LoginScreenComponent).toMatchSnapshot();
  });
});
