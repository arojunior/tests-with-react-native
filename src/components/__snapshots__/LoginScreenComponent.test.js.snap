// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreenComponent Snapshot 1`] = `[Function]`;
